packages/components/src/filter-picker/README.md

```diff
@@ -9,36 +9,44 @@ Modify a url query parameter via a dropdown selection of configurable options. T
 import { FilterPicker } from '@woocommerce/components';
 
 const renderFilterPicker = () => {
-	const filters = [
-		{ label: 'Breakfast', value: 'breakfast' },
-		{
-			label: 'Lunch',
-			value: 'lunch',
-			subFilters: [
-				{ label: 'Meat', value: 'meat', path: [ 'lunch' ] },
-				{ label: 'Vegan', value: 'vegan', path: [ 'lunch' ] },
-				{
-					label: 'Pescatarian',
-					value: 'fish',
-					path: [ 'lunch' ],
-					subFilters: [
-						{ label: 'Snapper', value: 'snapper', path: [ 'lunch', 'fish' ] },
-						{ label: 'Cod', value: 'cod', path: [ 'lunch', 'fish' ] },
-						// Specify a custom component to render (Work in Progress)
-						{
-							label: 'Other',
-							value: 'other_fish',
-							path: [ 'lunch', 'fish' ],
-							component: 'OtherFish',
-						},
-					],
-				},
-			],
-		},
-		{ label: 'Dinner', value: 'dinner' },
-	];
+        const config = {
```

Comment:
Hey, @manjeet-wisetr really appreciate your effort to better WooCommerce Admin! Sorry I'd have to make another nitpicky comment as well, our standard code style uses tabs instead of spaces, and it also seems that your changes use 8 spaces (usually its 4 spaces = 1 tab).

Reply:
@ilyasfoo I have made the changes, kindly review now. Thanks!!